coding-concepts.js

```diff
@@ -10,42 +10,40 @@
 var cohort = "Echo 2021"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "cohort.lenght" will produce the string's number of characters.
+// b) Verify and explain: To input, i used the node coding-concepts.js outputing 9, the number of characters in the "Echo_2021" string.
 
 
 // --------------------2) What will this log?
 
 var greeting = "Hello World!"
 // console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I believe it will do someting with [3] in "Hello World" or ?
+// b) Verify and explain: the output was "l" which makes me assume that [3] has to do with the 3rd letter, character, and/or index.
 
 
 // --------------------3) What will this log?
 
-var languages = ["JavaScript", "Ruby", "Python", "C++"]
-var index = 1
+// var languages = ["JavaScript", "Ruby", "Python", "C++"]
+// var index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The output would be "Ruby" as for "Javasript" is 0.
+// b) Verify and explain: The output is 1. 'Javascript'= 0, 'Ruby'=1
 
 
 // --------------------4) What will this log?
 
 var weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
-
+console.log(weekendDays.toUpperCase())
 
-// --------------------5) What will this log?
+// a) Your answer: What my understanding tells me, (never seen this command), is that ".toUpperCase" could execute the command to alternate the strings "saturday" & "sunday" letter case from lower case to upper case.
+// b) Verify and explain: I received an error from the Terminal.  My first troubleshoting steps was to commnet all commands.  Another Error.  Next, was to check for a typo, maybe done by me by accidentaly changing the original text, or done Bby LEARN purposely to assess my progress.
```

Comment:
Yup this returns an error. The method `.toUpperCase` works on strings. So when called on an array you get a Type Error.